pallets/handles/README.md

```diff
@@ -7,25 +7,56 @@ Creates human-readable, homoglyph-attack resistant handles for MSAs.
 Provides MSAs with an optional, but unique handle.
 
 A handle consists of:
-- **Base Handle:** The user's chosen handle. It is *not* guaranteed to be unique without the suffix. It is linked to a normalized version for Handle to MSA Id resolution. See [UTF-8 Support](#utf-8-support) and [Homoglyph Attack Resistence](#homoglyph-attack-resistence) below.
+- **Base Handle:** The user's chosen handle. It is *not* guaranteed to be unique without the suffix. It is linked to a normalized version for Handle to MSA Id resolution. See [Normalization Details](#normalization-details) below.
 - **Suffix:** A suffix is a unique numeric value appended to a user's base handle to make it unique.
 - **Display Handle:** The user's original (un-normalized, but with whitespace trimmed and consolidated) base handle string and the suffix together (`base`.`suffix`) constitute a unique identifier for a user.
 
-### UTF-8 Support
+### Suffixes
 
-Handles are able to have many allowed ranges of UTF-8.
-Some ranges, such as emoji, are currently not allowed.
-Due to the handling of homoglyphs, some handles will resolve to the same value.
-For example, while the display may have diacriticals or homoglyphs, the handle is stored with diacriticals and homoglyphs normalized.
-So `Zoë.35` and `Zoe.35` will both resolve to the same MSA Id.
+TO allow multiple users to select the same handle, a distinguishing unique suffix is appended to to form the Display Handle.
```

Comment:
In order to allow multiple users to select the same Base Handle, a unique numeric Suffix is appended to the Base Handle to form the Display Handle.

Reply:
```suggestion
In order to allow multiple users to select the same base handle, a unique numeric suffix is appended to the Base Handle to form the Display Handle.
```